common/src/app/common/geom/shapes/bool.cljc

```diff
@@ -6,21 +6,27 @@
 
 (ns app.common.geom.shapes.bool
   (:require
+   [app.common.data :as d]
    [app.common.geom.shapes.path :as gsp]
    [app.common.geom.shapes.rect :as gpr]
    [app.common.geom.shapes.transforms :as gtr]
    [app.common.path.bool :as pb]
    [app.common.path.shapes-to-path :as stp]))
 
+(defn calc-bool-content
+  [shape objects]
+  (->> (:shapes shape)
```

Comment:
I think this can be implemented using transducers

Reply:
OK